packages/back-end/src/services/clickhouse.ts

```diff
@@ -49,7 +49,7 @@ export async function createClickhouseUser(
   });
 
   const orgId = context.org.id;
-  const user = orgId;
+  const user = orgId + "ter";
```

Comment:
What does ter stand for? I could see maybe prefixing it with "user_", but it would be easier to copy and paste if we just left it as orgId. That might be useful in debugging or looking into an issue for a specific org.

Reply:
That was an old diff.